src/card/CardReducer.ts

```diff
@@ -22,6 +23,7 @@ const update = (state, mutations) => Object.assign({}, state, mutations);
  */
 
 export const CARD_INITIAL_STATE = {
+  index: createUUID(),
```

Comment:
i'd call it `id` instead of `index` here

Reply:
Also refactor in `CardSelectors`, `CardThunks`

Reply:
Done